composer.json

```diff
@@ -6,21 +6,21 @@
     "require": {
         "php": "^7.2",
         "sulu/sulu": "^2.0",
-        "ramsey/uuid": "^3.5"
+        "ramsey/uuid": "^3.1"
```

Comment:
@luca-rath can you define here all used Symfony components (range `^4.3 || ^5.0`) atleast here should be `config` and `dependency-injection`

Reply:
Also `http-foundation`, `http-kernel` and `routing` I think is needed